simpledotorg/simple-server
# Add BS measurement details

**Story card:** [sc-7784](https://app.shortcut.com/simpledotorg/story/7784/add-bs-measurement-details-card)

## Because

We need to add a graph showing the BS measurement details

## This addresses

Adds HTML and javascript changes necessary to show the breakdown graph

## Test instructions

1. Turn on diabetes_management_reports on the flipper
2. Visit the reports page of a facility which has enable_diabetes_management set to true
3. Click on the Diabetes link
4. You should see the BS measure details graph

See the screenshots for reference:
1.
<img width="1344" alt="Screenshot 2022-05-12 at 8 24 53 PM" src="https://user-images.githubusercontent.com/32141642/168104967-68a1aea0-833e-47ba-92f0-e2130f24ef96.png">
2.
<img width="1344" alt="Screenshot 2022-05-12 at 8 24 41 PM" src="https://user-images.githubusercontent.com/32141642/168104974-4d417d27-8b15-4eeb-8d1a-9402e25d48dd.png">
3.
<img width="1344" alt="Screenshot 2022-05-12 at 8 25 15 PM" src="https://user-images.githubusercontent.com/32141642/168104921-acc14e9b-7006-4515-8e1c-a0e230d7641c.png">
